src/Symfony/Component/Security/Acl/Tests/Dbal/MutableAclProviderTest.php

```diff
@@ -401,17 +402,25 @@ public function testUpdateAclDeletingObjectFieldAcesThrowsDBConstraintViolations
         $acl->insertObjectFieldAce($fieldName, $sid1, 4);
         $provider->updateAcl($acl);
 
+        $this->assertCount(1, $provider->findAcl($oid)->getObjectFieldAces($fieldName));
+
         $acl = $provider->findAcl($oid);
         $acl->insertObjectFieldAce($fieldName, $sid2, 4);
         $provider->updateAcl($acl);
 
+        $this->assertCount(2, $provider->findAcl($oid)->getObjectFieldAces($fieldName));
+
         $index = 0;
         $acl->deleteObjectFieldAce($index, $fieldName);
         $provider->updateAcl($acl);
 
+        $this->assertCount(1, $provider->findAcl($oid)->getObjectFieldAces($fieldName));
```

Comment:
shouldn't this be split into multiple tests instead ? Having assertions at 4 different points of the test looks weird.